### ISSUE TITLE
Liquid pro site down

### ISSUE THREAD

**Issue by Rickyboy320** [jljr222/xenforo-simple-forms#5]
Hello,

Suddenly our form functions of our forum stopped working. I noticed that there was an issue with liquidpro trying to fetch data. I tried to manually type in the site in my browser and it appeared I can no longer connect to the site. I'm sure you guys are working hard on getting it back up, but I'm wondering if I could get any update on the situation.

Thanks in advance.
Rick

**Comment by jljr222** [comment]
Hi Rick,

This is not an issue, the site is no longer accessible because we are no longer in business. Per our email we sent out some time ago, we've decided to close shop for personal reasons. We've released the addon publically here for those who wish to download and use. We've also had a few contributors make updates and fixes to the addon.

https://github.com/Xon/XenForo-SimpleForms/archive/2.1.9f.zip

Download that version of Simple Forms and upgrade over your current installation. All should work as expected.
